yaki_backend/README.md

```diff
@@ -0,0 +1,44 @@
+# node-server
+this is nodeJs api YAKI
+
+## Purpose
+This project is the backend part of the YAKI application.
+
+## General info
+### Authentification
+No GAFAM will be used for authentification
+We will implement a home made authentification process
+We will use JWT authorization token in the Authorization header
+
+### Data format
+- teamMate : id, firstName, lastName, email
+- captain: id, firstName, lastName, email
+- team: id, idCapitain, name
+- declaration: id, idTeamMate, status, date
+
+### Path
+- GET /teamMates => get all team-mates
+- GET /teamMates/{id} => get a team-mate by id
+- GET /teamMates/{id}/declarations => get all team-mate declarations 
+- GET /captains => get all captains
+- GET /captains/{id} => get a captain by id
+- GET /captains/{id}/teams => get all captain teams
```

Comment:
According to the screens we should have at least operations to:

- login
- logout
- GET teammate today's, morning, afternoon declarations
- POST teammate new declaration
- GET captain teams
- GET captain team members
- GET captain team members declarations

more operations will be added in future versions